src/main/java/com/lithium/dbi/rdbi/recipes/scheduler/ExclusiveJobSchedulerDAO.java

```diff
@@ -71,7 +71,7 @@ int scheduleJob(
     @Query(
         "local deletedFromReadyQueue = redis.call('ZREM', $readyQueue$, $job$)\n" +
         "local deletedFromRunningQueue = redis.call('ZREM', $runningQueue$, $job$)\n" +
-        "if not deletedFromReadyQueue and not deletedFromRunningQueue then\n" +
+        "if deletedFromReadyQueue == 0 and deletedFromRunningQueue == 0 then\n" +
```

Comment:
you sure? http://www.lua.org/pil/2.2.html

"Beware that, unlike some other scripting languages, Lua considers both zero and the empty string as true in conditional tests."

Was this a bug in the original impl ?